### ISSUE TITLE
set_view_order doesn't work as it should. pulls view args up into top level

### ISSUE THREAD

**Issue by jordansread** [USGS-R/gsplot#224]
Noticed this when working on #193 

``` R
gs <- gsplot()  %>%
  points(1:10, col="black")  %>%
  bgCol(col="coral2")
#then putting a breakpoint on the return of set_view_order:
names(append(dataViews, otherViews))
[1] "view"   "window" "points"
#when points shouldn't be moved up to high level (window either). 
```

**Comment by ldecicco-USGS** [comment]
So what's going on here?

```
gs <- gsplot()  %>%
     points(1:10)
gs
```

We lost the ability to only specify 1 vector it seems.

**Comment by jordansread** [comment]
@ldecicco-USGS see #245. oh yes, you saw it

**Comment by jordansread** [comment]
no, that example is an old version, but the behavior on v 0.3+ is the same

**Comment by jordansread** [comment]
ignore the y=NULL issue, that is different

**Comment by lindsayplatt** [comment]
@jread-usgs @ldecicco-USGS I don't think this issue is relevant anymore. Can someone look and then close if you agree?

**Comment by lindsayplatt** [comment]
There appear to be two issues mentioned here: 
1. Points being top-level. This is no longer an issue:
```r
gs <- gsplot()  %>%
  points(1:10, col="black")  %>%
  bgCol(col="coral2")
names(gs)
[1] "side.1"   "side.2"   "global"   "view.1.2"
```
2. Losing the ability to only specify one vector of points, which is being addressed in #245 .

@ldecicco-USGS and @jread-usgs please close if you agree that this issue is no longer relevant.
